beacon-chain/state/stateAltair/BUILD.bazel

```diff
@@ -3,19 +3,27 @@ load("@prysm//tools/go:def.bzl", "go_library")
 go_library(
```

Comment:
it's nonstandard to use camel case for directory naming. We should probably call it altair as a package perhaps?